mempool/src/shared_mempool.rs

```diff
@@ -359,8 +490,13 @@ async fn gc_task(mempool: Arc<Mutex<CoreMempool>>, gc_interval_ms: u64) {
 pub(crate) fn start_shared_mempool<V>(
     config: &NodeConfig,
     mempool: Arc<Mutex<CoreMempool>>,
-    network_sender: MempoolNetworkSender,
-    network_events: MempoolNetworkEvents,
+    // (key, val) = (network ID, handle to network).
+    // See `NodeConfig::is_upstream_peer` for the definition of network ID
+    mempool_network_handles: HashMap<PeerId, MempoolNetworkHandle>,
+    ac_endpoint_receiver: mpsc::Receiver<(
```

Comment:
can we unify naming of this parameter across different paths?
and remove "ac" part from it
 `client_events` maybe?